src/components/notebook-utils.js

```diff
@@ -48,8 +48,18 @@ export const notebookNameValidator = existing => ({
     message: 'already exists'
   }
 })
+// removes leading dirs and a file ext suffix on paths
+export const getDisplayName = name => stripExtension(getFileName(name))
 
-export const notebookNameInput = ({ inputProps, ...props }) => h(ValidatedInput, {
+// removes all paths up to and including the last slash
+export const getFileName = path => path.split('/').pop()
+
+// removes a file extension from a string
+export const stripExtension = _.replace(/\.[^/.]+$/, '')
+
+export const getExtension = name => name.split('.').pop()
```

Comment:
To avoid mutating anything, and for consistency, consider this:
```suggestion
export const getExtension = _.flow(_.split('.'), _.last)
```